image-trigger

```diff
@@ -66,6 +66,8 @@ REFRESH = {
     "rhel-9-4": {},
     "rhel4edge": {},
     "services": REFRESH_30,
+    "opensuse-tumbleweed": REFRESH_30,
```

Comment:
Tumbleweed is rolling release right? So then I would say it should refresh as often as the rest of our images.

Reply:
how would you suggest i do this? just set it to {}?

Reply:
Yes, `{}` means default parameters which is 7 days.